### PR TITLE
Issue154 change password page

### LINKED ISSUE #154: FRONTEND: Create the Change Password page
Page 56 of the mockups https://drive.google.com/file/d/1YqboQogczYm1HkyRqEtVSzeQ61T9hWU2/view

### PR DESCRIPTION
**Pull request by Chris-Boe**
### What issue is this solving?
Issue #154 

### Any helpful knowledge/context for the reviewer?
Is a `npm install` necessary? Nope
Any special requirements to test? Just change your password on that page, log out, then login again with the new password after verifying the old password no longer works.

### Please make sure you've attempted to meet the following coding standards
- [x] Code builds successfully
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code
- [x] There aren't any unnecessary commits or files changed (shouldn't touch the /stashed folder unless the issue requests it)

If you're having trouble meeting this criteria, feel free to reach out on Slack for help!

### CODE REVIEW

File: src/views/Settings/changePassword.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
+import React, { useContext } from "react";
+import { Link, useHistory } from "react-router-dom";
+import { Form, Field, Formik } from "formik";
+import * as Yup from "yup";
+import * as axios from "axios";
+import UserContext from "../../UserContext";
+import Button from "../../components/Button";
+
+const validationSchema = Yup.object().shape({
```
Comment (NickSchimek) [inline src/views/Settings/changePassword.js:9]: I guess all the good names are taken? `Yup`

File: src/views/Settings/changePassword.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        password: data.new
+      }, {
+        headers: { Authorization: `Bearer ${context.user.accessJwt}` },
+      })
+      .then((response) => {
+        // once Toast is implemented, replace with Toast notification
+        alert("Saved Successfully!");
+      })
+      .then(() => {
+        history.push("/settings");
```
Comment (NickSchimek) [inline src/views/Settings/changePassword.js:36]: What is the purpose of this line here?

**Review by NickSchimek — APPROVED** [review]
Looks good. You probably want to get another eye on this as I can't say much about the code itself. However, I verified through the UI that everything looks good and works as expected.

Although I did notice one issue, a backend issue, the user does not have to input the correct current password in order to change their password.  Created issue #340 to address this.

File: src/views/Settings/changePassword.js (hunk 2 above)
Comment (Chris-Boe) [reply src/views/Settings/changePassword.js:36]: This moves the user back to the settings page after they've changed their password and clicked "Ok" on the alert from line 33

**Pull request merged**